contrib/k8s.io/client-go/kubernetes/example_test.go

```diff
@@ -29,7 +30,7 @@ func Example() {
 		panic(err.Error())
 	}
 
-	pods, err := client.CoreV1().Pods("default").List(meta_v1.ListOptions{})
```

Comment:
Was this a breaking change we'll need to notify about during the release? (technically it's not _our_ break since it's the underlying library, but still it's a bit odd to upgrade a minor version of dd-trace-go and get a breaking build because the underlying `k8s.io/client-go` is major version 0 and able to break whenever it wants)

Reply:
Updated the PR title accordingly so we mention it in the release note.